script.js

```diff
@@ -1,4 +1,101 @@
+var mode = "player1mode";
+var myOutputValue = "";
+var p1Results = "";
+var p2Results = "";
+var dice1 = "";
+var dice2 = "";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  //player 1
+  //when player 1 clicks submit
+  if (mode == `player1mode`) {
+    // generate random dice number
+    dice1 = random();
+    dice2 = random();
+    console.log(dice1);
+    console.log(dice2);
+
+    myOutputValue = `Welcome Player 1. <br> You rolled ${dice1} for Dice 1 and ${dice2} for Dice 2. <br> Choose the order of the dice. <br>
+    Enter A to select ${dice1}${dice2} or B to select ${dice2}${dice1}.`;
+    // change to choice mode
+    mode = `player1Choicemode`;
+  }
+  // player 1 choses order
+  else if (mode == "player1Choicemode") {
+    // if player 1 chooses A
+    if (input == "A" || input == "a") {
+      p1Results = dice1 * 10 + dice2;
+      console.log("Player 1 chose A");
+      myOutputValue = `Player 1, you have selected option A: ${p1Results}. Your Current score is ${p1Results}. <br> Player 2's turn now.`;
+      mode = `player2mode`;
+    }
+    // if player 1 chooses B
+    else if (input == "B" || input == "b") {
+      p1Results = dice2 * 10 + dice1;
```

Comment:
this is a really interesting way of coding your logic. how would you implement this if there were 6 dice instead of 2?